hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -53,7 +53,7 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
             sanatizeInstitutionIdentifiers(institution, true, uri, postInstitution)
           } ~
             timedPut { uri =>
-              sanatizeInstitutionIdentifiers(institution, false, uri, putInstitution)
+              sanatizeInstitutionIdentifiers(institution, true, uri, putInstitution)
```

Comment:
@kgudel / @Kibrael  was the logic behind this due to the case in which there might be an institution with an LEI of an invalid length already in the ecosystem? If so I can adjust this so on `PUT` we can check the LEI only when not found and creating a new institution.